### ISSUE TITLE
Apps in W10 Start menu remain after a distro is unregistered

### ISSUE THREAD

**Issue by EricPell** [microsoft/wslg#24]
Fairly self explanatory.  

1. Export a disto, and reimport it to Ubuntu_number2
2. Install gui app, like emacs in Ubuntu_number2
3. Unregister with `wsl --unregister Ubuntu_number2`
4. Items in start menu remain, with entry for Ubuntu_number2

**Comment by hideyukn88** [comment]
@EricPell, yes, this is current limitation, as workaround, please remove them manually from %APPDATA%\Roaming\Microsoft\Windows\Start Menu\Programs and icon data from %TMP%\WSLDVCPlugin.

**Comment by hideyukn88** [comment]
Fixed in WSLg 1.0.39. There is some small issue, if distro is uninstalled from start menu (right click on distro icon, then uninstall) the menu items will be removed when next WSL launch. If distro is uninstalled by `wsl --unregister`, then it will be removed immediately.
